src/Microsoft.Health.Fhir.Core/Microsoft.Health.Fhir.Core.csproj

```diff
@@ -37,6 +37,8 @@
     <PackageReference Include="Microsoft.Health.Extensions.DependencyInjection" Version="1.0.0-master-20201102-1" />
     <PackageReference Include="Hl7.Fhir.Serialization" Version="1.9.0" />
     <PackageReference Include="Hl7.FhirPath" Version="1.9.0" />
+    <PackageReference Include="Microsoft.Health.Fhir.Liquid.Converter" Version="1.0.0-master-20201116-2" />
```

Comment:
Wondering if we should move this out of the Core project (cc @brendankowitz)

Reply:
Thoughts on starting to move to a `Microsoft.Health.Fhir.Extensions.DataConverter` strategy for adding components? @johnstairs I agree that this isn't really part of core FHIR logic

Reply:
So you mean we should create a new extension project or a project for data convert?

Reply:
As we discussed offline, the refactoring will happen after this PR (Task to track here https://microsofthealth.visualstudio.com/Health/_workitems/edit/77677 )